src/workspaces/actions.ts

```diff
@@ -51,6 +51,7 @@ const WorkspacesActions = {
     'workspaces/addVocabularySuccess',
     'workspaces/addVocabularyFailure'
   )<AddVocabularyPayload, AddVocabularyPayload, Error>(),
+  openInTool: createAction('workspaces/openInToolRequest' )<void>(),
```

Comment:
I'd consider just using the name `workspaces/openInTool`, as the `Request` suffix is usually reserved for the async actions. Also the `<void>` part can be omitted.